src/commands/commands/agile/commands/work/commands/create/agileWorkCreate.ts

```diff
@@ -19,8 +21,14 @@ export const agileWorkCreate = new Command()
     }
 
     // Read agile items from provided file
-    const { file } = options;
-    const template: BacklogItemTemplate = await FileUtils.readJson(file || FileConstants.backlogItemsFileName);
+    const { file, params: paramsPath } = options;
+    const templatePath = file || FileConstants.backlogItemsFileName;
+
+    const parameters = FileUtils.readJson<Parameters>(paramsPath);
+    const template = FileUtils.readJson<BacklogItemTemplate>(templatePath, parameters);
+    if (!template) {
+      throw new Error(`Could not find template '${templatePath}'`);
```

Comment:
Should we add a test to make sure it throws the error when reading the file fails?

Reply:
Will do